models/pendulum/scene.xml

```diff
@@ -0,0 +1,25 @@
+<mujoco model="pendulum_scene">
```

Comment:
Nothing looks wrong with this, but would it be possible to dev on a branch instead of a fork? If you're on a branch, it makes it easy for me to `fetch` your changes and visually inspect the scene associated with this file. It's cumbersome to pull a fork and also maintain the main codebase just to review a small PR.

Reply:
You should be able to `fetch` from a fork just like you would from a branch. Just add the fork as a new remote, e.g., `git remote add [remote-name] https://github.com/vincekurtz/ambersim`. The syntax just becomes something like `git pull [remote-name] [branch-name]` instead of `git pull origin [branch-name]`. Other than the one-time `remote add` I don't think that's any more cumbersome? Unless I'm missing something?

Reply:
I'm embarrassed that I didn't know how to do this. Thanks.